DEFRA/ffc-sfi-agreement-calculator
# Publish eligibility events

- Version bump
- add rules
- Change to checkEligibility function
- Update npm
- version bump
- Add tests
- Add eligibility tests
- Update test details
- Update test values
- Filter non address line
- Break out of loops once area met
- Chane to getLandcover
- Updates after review
- Removed redunent requires
- updated return
- Changes and added tests
- Update version
- Initial test added
- Add eligiblity check
- Adapter updated
- Update Service Bus adapter
- fix lock
- Fix lock
- Replicas changed
- AsyncAPI spec added
- Add eligibility events
